### ISSUE TITLE
UnityEvent.emit(0) fails

### ISSUE THREAD

**Issue by GregoryPotdevin** [jeffreylanters/react-unity-webgl#24]
I have a method passing array indexes from React to Unity code, and the code works for all values but 0.

It looks like there's a bug when trying to set a default value :
https://github.com/jeffreylanters/react-unity-webgl/blob/master/source/modules/UnityEvent.js#L13
```js
parameter || ''
```

Falsy values are always replaced by an empty string.

**Comment by jeffreylanters** [comment]
Hi, I'm not at my mac at the moment.
If you want to you can create a pull request and replace 
```cs
parameter || ''
```
with
```cs
typeof parameter !== 'undefined' ? parameter || ''
```
and compile it, and can merge it. Otherwise you'll have to wait until tonight.

**Comment by GregoryPotdevin** [comment]
No rush, I worked around as I'm just doing a quick POC for now. If I get to work on this project some more I'll gladly contribute if I see other issues/enhancements.

**Comment by jeffreylanters** [comment]
https://github.com/jeffreylanters/react-unity-webgl/pull/25

**Comment by jeffreylanters** [comment]
Fixed in 6.4.2
